lib/phoenix_component/declarative.ex

```diff
@@ -746,7 +863,10 @@ defmodule Phoenix.Component.Declarative do
         build_attr_name(slot_attr),
         build_attr_type(slot_attr),
         build_attr_required(slot_attr),
-        build_attr_doc_and_default(slot_attr, "    ")
+        build_hyphen(slot_attr),
+        build_attr_doc_and_default(slot_attr, "    "),
+        build_attr_values(slot_attr),
+        build_attr_examples(slot_attr)
```

Comment:
We can probably unify it as:

```suggestion
        build_attr_values_or_examples(slot_attr)
```

Given both cannot be set. :)